README.md

```diff
@@ -139,13 +139,8 @@ There is `Vagrantfile` provided with braid, that will setup a staging sever.
 It uses the address `172.16.255.140`, and there is a braid config named `vagrant` that connects to it by default.
 
 ```shell
-# Get the required base OS image if you don't have it yet.
-vagrant box add precise64 http://files.vagrantup.com/precise64.box
```

Comment:
I added that line because on my case `vagrant up` failed to get the pricese image and I had a hard time finding the url.

here is the result of vagrant up from my Ubuntu 14.04

```
$ vagrant up
Bringing machine 'default' up with 'virtualbox' provider...
There are errors in the configuration of this machine. Please fix
the following errors and try again:

vm:
* The box 'ubuntu/precise64' could not be found.

$ vagrant --version
Vagrant 1.4.3
```

---

and what about the other documentation which was removed?